src/Modules/Shared/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psm1

```diff
@@ -737,4 +737,18 @@ function ConvertFrom-SddlString
             RawDescriptor = $rawSecurityDescriptor
         }
     }
-}
\ No newline at end of file
+}
+
+function Get-Uptime
+{
+    [CmdletBinding()]
+    [Alias("uptime")]
+    [OutputType([System.Timespan])]
+    Param()
+
+    Process
+    {
+        # It is portable
+        [TimeSpan]::FromSeconds([system.diagnostics.stopwatch]::GetTimestamp()/[system.diagnostics.stopwatch]::Frequency)

```

Comment:
>GetTimestamp [](start = 64, length = 12)

I believe this API is hardware dependent - if `IsHighResolution` false, you'll get back `[datetime]::now.ticks` which won't give the proper answers. You might want to be checking for that. something like:
```
if ( [system.diagnostics.stopwatch]::IsHighResolution) 
{
[TimeSpan]::FromSeconds([system.diagnostics.stopwatch]::GetTimestamp()/[system.diagnostics.stopwatch]::Frequency)
}
. . .
```